src/Hosting/samples/GenericHostSample/GenericHostSample.csproj

```diff
@@ -2,7 +2,6 @@
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.0;net461</TargetFrameworks>
-    <RuntimeIdentifiers>win7-x64;linux-x64;osx-x64</RuntimeIdentifiers>
```

Comment:
Why? These were useful for testing the windows service.

Reply:
Maybe, they're useful for net461 maybe but we now make native exes by default. I was also just trying to see if this would fix the issue I was running into but it didn't.

Reply:
I was using them for standalone publish.

Reply:
Why? You *had* to do that in the past to get an exe, not you don't. What other reason did you have to self contain deploying this app?

Reply:
Services run as different users without access to the repo local runtime. It's easier to deploy the runtime with the service than to tell the service how to find it.

Reply:
You can publish self contained without putting RIDS in the prodject file and doing a RID restore. That's been the case since 2.x.